app/scripts/services/udbhttpinterceptor.js

```diff
@@ -30,7 +30,7 @@ function httpInterceptor($q, $location, $window, appConfig, $translate) {
         return $q.reject(rejection);
       }
 
-      if (rejection.status === 403 && rejection.title === 'Label not allowed') {
+      if (rejection.status === 403 && (rejection.data && rejection.data.title === 'Label not allowed')) {
```

Comment:
Did the returned data change? or was this always a bug? or can both `rejection.title` and `rejection.data.title` be defined?

Reply:
To me it should always be `.data` since it's an object response, so I indeed wondered if it was always broken especially since the translation for that error was used nowhere up until now. Perhaps it used to be caught and shown and then was changed? 🤔

Reply:
I've checked the original PR and it was for the same feature in the detail pages https://github.com/cultuurnet/udb3-angular-app/commit/b35ae211b55fa35fdd934e2802fede55f986721d but I've checked with the new code and the detail pages seem to show the error label correctly, and logging the rejection does indeed not contain a `title` field so maybe we changed how the rejections are transformed somewhere 🤔

Reply:
Ok 👍🏻